src/sparse_blas/backends/mkl_common/mkl_handles.cxx

```diff
@@ -369,7 +282,7 @@ sycl::event release_sparse_matrix(sycl::queue &queue, oneapi::mkl::sparse::matri
     // Asynchronously release the backend's handle followed by the internal handle.
     auto event = oneapi::mkl::sparse::release_matrix_handle(
         queue, &internal_smhandle->backend_handle, dependencies);
-    return detail::submit_release(queue, internal_smhandle, event);
+    return detail::submit_release(queue, internal_smhandle, { event });
```

Comment:
Hmm, how was this working before? Did we have a `sycl::event` arg in `submit_release()` before? Have you checked this change still compiles and works for all backends? I may be missing something here.

Reply:
The previous version of `submit_release` was in [`generic_container.hpp`](https://github.com/oneapi-src/oneMKL/pull/527/files#diff-94c98d6e052fa1c50afe95a3973550ec75c0a51fcdea2d80b5de68886c9f83f6L325) and had a template argument to accept both single event or a vector of events. I've not kept this template argument after moving  `submit_release` since only `release_sparse_matrix` for the MKL backends used it. I doubt it makes any measurable difference.
I test the PR with the MKL backends after large changes. The current version ran fine. I'll make sure to attach recent logs next time I test.